### PR TITLE
Change songindex to index, switch icon to fa-list

### PR DESCRIPTION
**Pull request by jithatsonei**
This proposal comes as with the implement of tags the songindex page has morphed to more then just that.

### CODE REVIEW

**Comment by jkhsjdhjs** [comment]
good idea imo, cause not every webm has a song
i think it was named songindex cause z0r has it like this, but on z0r nearly every video has a song

File: resources/views/partials/navigation.blade.php [hunk 1]
```diff
@@ -13,7 +13,7 @@
         <div class=" collapse navbar-collapse" id="bs-example-navbar-collapse-1">
             <ul class="nav navbar-nav navbar-left">
                 <li><a href="{{url('categories')}}"><i class="fa fa-bars"></i> Categories</a></li>
-                <li><a href="{{url('songindex')}}"><i class="fa fa-music"></i> Songindex</a></li>
+                <li><a href="{{url('songindex')}}"><i class="fa fa-list"></i> Index</a></li>
```
Comment (GottZ) [inline resources/views/partials/navigation.blade.php:16]: i think url should be changed aswell

**Comment by jithatsonei** [comment]
will do tomorrow

**Comment by jithatsonei** [comment]
closing pr in favor of new one

**Pull request closed without merging**